system-linux.c

```diff
@@ -2074,6 +2091,7 @@ system_if_apply_settings(struct device *dev, struct device_settings *s, uint64_t
 	if (apply_mask & DEV_OPT_ARP_ACCEPT)
 		system_set_arp_accept(dev, s->arp_accept ? "1" : "0");
 	system_set_ethtool_settings(dev, s);
+	system_set_ethtool_eee_settings(dev, s);
```

Comment:
I would put this in the ethtool_settings function down the pause settings. And also apply only if the flag is applied... leaving the default value. Currently this will always disable eee if not set.

Reply:
thanks for your review @Ansuel , I have updated the code and the commit message and also rebased the code.